examples/docs/src/components/side_bar.rs

```diff
@@ -30,32 +30,32 @@ pub fn SideBar(router: Router) -> Dom {
                 <li>
                   <span class="font-medium">"Inbuilt Mixins"</span>
                   <ul class="pl-3">
-                    <li class="">
+                    <li class="mixin-s-link">
```

Comment:
Not sure what these classes are for. Did you add some styling?

Reply:
No I did not, will delete it.